### ISSUE TITLE
Empty error message

### ISSUE THREAD

**Issue by f100024** [cshum/imagor#301]
Hello,
I faced with next issue during resizing the image with request:
```
http://127.0.0.1:8000/unsafe/trim/fit-in/0x200/filters:sharpen(1,0,true)/d4b6245b-99e1-49cd-8fad-2b40b8eacca9.jpg
```
```
2023-01-16T16:40:28.094Z        WARN    imagor/imagor.go:407    process {"params": {"path":"trim/fit-in/0x200/filters:sharpen(1,0,true)/d4b6245b-99e1-49cd-8fad-2b40b8eacca9.jpg","image":"d4b6245b-99e1-49cd-8fad-2b40b8eacca9.jpg","unsafe":true,"trim":true,"trim_by":"top-left","fit_in":true,"height":200,"filters":[{"name":"sharpen","args":"1,0,true"}]}, "error": "imagor: 406 "}
github.com/cshum/imagor.(*Imagor).Do.func2
        /go/src/github.com/cshum/imagor/imagor.go:407
github.com/cshum/imagor.(*Imagor).suppress.func2
        /go/src/github.com/cshum/imagor/imagor.go:611
golang.org/x/sync/singleflight.(*Group).doCall.func2
        /go/pkg/mod/golang.org/x/sync@v0.1.0/singleflight/singleflight.go:189
golang.org/x/sync/singleflight.(*Group).doCall
        /go/pkg/mod/golang.org/x/sync@v0.1.0/singleflight/singleflight.go:191
```
`Imagor` returned an empty error message, but vips [here](https://github.com/cshum/imagor/blob/master/vips/foreign.c#L22) returned `1`:
```
...
  if (vips_cache_operation_buildp(&operation)) {
    vips_object_unref_outputs(VIPS_OBJECT(operation));
    g_object_unref(operation);
    return 1;
  }
...
```
`Thumbor` worked well with this image and I don't see any errors in this file.
What can be wrong with the image?

Test image in the archive: 
[d4b6245b-99e1-49cd-8fad-2b40b8eacca9.zip](https://github.com/cshum/imagor/files/10427856/d4b6245b-99e1-49cd-8fad-2b40b8eacca9.zip)

**Comment by f100024** [comment]
Looks like fixed. Resizes as expected.
BTW, I applied `libvips` filters from examples, it also worked without errors.
Thank you!

**Comment by cshum** [comment]
Available in v1.3.7
